client/settings/general-settings-section/__tests__/remove-method-confirmation-modal.test.js

```diff
@@ -4,8 +4,13 @@ import userEvent from '@testing-library/user-event';
 import RemoveMethodConfirmationModal from '../remove-method-confirmation-modal';
 
 describe( 'RemoveMethodConfirmationModal', () => {
-	const handleCloseMock = jest.fn();
-	const handleRemoveMock = jest.fn();
+	let handleCloseMock;
+	let handleRemoveMock;
+
+	beforeEach( () => {
+		handleCloseMock = jest.fn();
+		handleRemoveMock = jest.fn();
+	} );
```

Comment:
I feel like this change would only slow down the tests. Is there a reason this change is necessary? 🤔